File: archives/10214.py
T = int(input())
for _ in range(T):
	result = 0
	yonsei, korea = 0, 0
	for _ in range(9):
		Y, K = map(int, input().split())
		yonsei += Y
		korea += K

	if yonsei > korea:
		print("Yonsei")
	elif yonsei < korea:
		print("Korea")
	else:
		print("Draw")

File: archives/cpp/10214.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    cout.tie(NULL);
    
    int T; cin >> T;
    int Y = 0, K = 0;
    int y, k;

    while (T--) {
        for (int i = 0; i < 9; i++) {
            cin >> y >> k;
            Y += y; K += k;
        }

        if (Y > K) { cout << "Yonsei\n"; }
        else if (Y < K) { cout << "Korea\n"; }
        else { cout << "Draw\n"; }
    }

    return 0;
}